### PR TITLE
assigns file owner to attached files on work creation

### PR DESCRIPTION
**Pull request by lawhorkl**

### CODE REVIEW

File: app/services/curation_concern/generic_work_actor.rb [hunk 1]
```diff
@@ -123,6 +123,7 @@ def attach_file(file)
       )
       generic_file.embargo_release_date = curation_concern.embargo_release_date
       generic_file.visibility = visibility
+      generic_file.owner = user.user_key
```
Comment (jamesvanmil) [inline app/services/curation_concern/generic_work_actor.rb:126]: The file owner isn't the current user, in the case of delegate submitted works.

Compare with https://github.com/uclibs/curate/blob/develop/app/services/curation_concern/base_actor.rb#L77

**Comment by lawhorkl** [comment]
I'll fix it up and push today during the long drive

**Comment by hortongn** [comment]
@lawhorkl what do you mean by "during the long drive?"  Will you not be in the office today?  Just curious.  I noticed your MacBook wasn't here, but wasn't sure if you were working from elsewhere in the library.

**Comment by hortongn** [comment]
Does this pull request still need a change made?  I don't think anything has been updated since James' comment above

**Comment by lawhorkl** [comment]
It is not ready for merging yet, still trying to resolve the error I'm getting.

Kyle Lawhorn
Software Development Co-Op
University of Cincinnati Libraries
CECH | IT:Cybersecurity
(513)370-6524 | lawhorkl@mail.uc.edu

**Pull request merged**